templates/shared/user/authorlink.tmpl

```diff
@@ -1 +1 @@
-<a class="author text black gt-font-semibold muted"{{if gt .ID 0}} href="{{.HomeLink}}"{{end}}>{{.GetDisplayName}}</a>
+<a class="author text black gt-font-semibold muted"{{if gt .ID 0}} href="{{.HomeLink}}"{{end}}>{{.GetDisplayName}}</a>{{if .IsBot}}<span class="ui label">bot</span>{{end}}
```

Comment:
Please add `bot` as translated key. Otherwise LGTM.
(Sidenote: I wonder in how many places this won't be called because the templates don't use this template yet. But that's no concern for this PR.)

Reply:
I didn't localize `bot` because passing `locale` into the `autorlink` template would require changing all calls. It caused many bugs when we did it for `avatarlink`